docs/Build.md

```diff
@@ -0,0 +1,72 @@
+Naggum Build Guide
+==================
+
+Windows
+-------
+
+To build Naggum under Windows, just use Visual Studio or MSBuild like that:
+
+    $ cd naggum
+    $ nuget restore
+    $ msbuild /p:Configuration=Release Naggum.sln
+
+Linux
+-----
+
+See general build instructions for Linux in the file `.travis.yml` inside of the
+Naggum root source directory.
+
+You'll need [Mono][mono], [NuGet][nuget] and [F# Compiler][fsharp] installed in
+you current environment. Some of them may or may not be part of your Mono
```

Comment:
My intention here was to clarify one noticeable thing. In NixOS you may install software to different environments/scopes (global, user-bound, shell-bound), and it'd be nice to say that we support any of these.

But now I agree with you, just "installed" should be enough; `nix-env` already installs software to the "current" environment (if the read cares about that at all).